### PR TITLE
Add function to convert underscored names to camelCased

### PR DESCRIPTION
**Pull request by ross-ritchey**
Added a function to camelCase an input

### CODE REVIEW

**Review by yithril — COMMENTED** [review]
Are we sure we need to ignore D store?

Look at lines 8 through 11 is that the most efficient?

File: src/utilities/Utilities.js [hunk 1]
```diff
@@ -2,3 +2,14 @@ export const capitalize = (s) => {
     if (typeof s !== "string") return "";
     return s.charAt(0).toUpperCase() + s.slice(1);
 };
+
+export const convertUnderscoresToCamelCase = (s) => {
+  if (typeof s !== "string") return "";
+  let sArr = string.split(/[_-]+/);
+  let retString = sArr.pop();
+  sArr.forEach((str) => {
```
Comment (yithril) [inline src/utilities/Utilities.js:10]: What is the Big O for this?

**Review by yithril — CHANGES_REQUESTED** [review]
So look at my comments I left in the file and please make changes

**Comment by ross-ritchey** [comment]
@yithril Please review again. I removed DS_Store from the gitignore.

The Big O for that is O(n) so it should be good.

**Review by yithril — APPROVED** [review]
LGTM!

**Pull request merged**